tests/files/locations/test_gcs.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from astro.files.locations import location_factory
+
+
+def describe_is_valid_path():
+    """test is_valid_path() method"""
+
+    def with_supported_filepaths():  # skipcq: PTC-W0065, PYL-W0612
+        """With supported file paths"""
+        filepath = "gs://bucket/some-file"
+        location = location_factory(filepath)
+        assert location.is_valid_path(filepath) is True
+
+
+def describe_get_transport_params():
+    """test get_transport_params() method"""
```

Comment:
Can you add comment about what we are trying to test here for example "Test get_transport_params returns a GCS client"

Reply:
Since `get_transport_params` inside `tests/files/locations/test_gcs.py` will only ever test gcs`, do we need to use describe?

we can just call it `test_get_transport_params` and since this is `test_gcs` file that is fine, wdyt?

Reply:
Totally, Done.